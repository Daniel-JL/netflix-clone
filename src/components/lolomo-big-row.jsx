import React from 'react';
import {
  NavLink,
} from 'react-router-dom';
import styled from 'styled-components';

const BigRowContainer = styled.div`
  width: 100%;
  padding-top: 41%;
  border: 1px solid black; 
`;

export function LolomoBigRow() {
  return (
    <BigRowContainer>

    </BigRowContainer>
  )
}