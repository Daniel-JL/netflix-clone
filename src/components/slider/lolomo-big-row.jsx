/* eslint-disable import/prefer-default-export */
import React from 'react';
import {
  Link,
} from 'react-router-dom';
import styled from 'styled-components';

const BigRowContainer = styled.div`
  width: 100%;
  padding-top: 41%;
  border: 1px solid black; 
`;

export function LolomoBigRow() {
  return (
    <BigRowContainer>

    </BigRowContainer>
  );
}
