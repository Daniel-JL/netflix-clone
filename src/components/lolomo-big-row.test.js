import React from 'react';
import { fireEvent, render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { LolomoBigRow } from './lolomo-big-row';

describe('Lolomo big row row component', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(<LolomoBigRow />).toJSON();

    expect(wrapper).toMatchSnapshot();
  });

  it('should contain a video player', () => {

  });

  it('should have a mute button on the video player', () => {

  });

  it('should show the maturity rating beside the mute button', () => {

  });

  it('should have two buttons and a description beside the video player', () => {

  });
});