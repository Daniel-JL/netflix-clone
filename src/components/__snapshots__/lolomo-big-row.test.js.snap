// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lolomo big row row component renders correctly 1`] = `
<div
  className="sc-AxjAm bbcUpW"
/>
`;
